### PR TITLE
feat: Implement Python virtual environment display

### PR DESCRIPTION
**Pull request by MaT1g3R**
#### Description

The prompt will now display the current Python virtual environment like this:
`foo on  master via 🐍 v3.7.4(bar)`

#### Motivation and Context

It is handy to show the active Python virtual environment in the prompt

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: src/modules/python.rs [hunk 1]
```diff
@@ -61,6 +66,14 @@ fn format_python_version(python_stdout: &str) -> String {
     format!("v{}", python_stdout.trim_start_matches("Python ").trim())
 }
 
+fn get_python_virtual_env() -> Option<String> {
+    env::var("VIRTUAL_ENV").ok().and_then(|venv| {
+        Path::new(&venv)
+            .file_name()
+            .map(|filaname| String::from(filaname.to_str().unwrap_or("")))
```
Comment (johnletey) [inline src/modules/python.rs]:
```suggestion
            .map(|filename| String::from(filename.to_str().unwrap_or("")))
```

File: tests/testsuite/python.rs [hunk 2]
```diff
@@ -71,3 +73,23 @@ fn folder_with_py_file() -> io::Result<()> {
     assert_eq!(expected, actual);
     Ok(())
 }
+
+#[test]
+#[ignore]
+fn with_virtual_env() -> io::Result<()> {
```
Comment (matchai) [inline tests/testsuite/python.rs]: 👍

File: src/modules/python.rs [hunk 3]
```diff
@@ -70,4 +83,16 @@ mod tests {
...
         assert_eq!(format_python_version(input), "v3.7.2");
     }
+
+    #[test]
+    fn test_no_virtual_env() {
+        env::set_var("VIRTUAL_ENV", "");
+        assert_eq!(get_python_virtual_env(), None)
+    }
+
+    #[test]
```
Comment (matchai) [inline src/modules/python.rs]: 👍

File: docs/config/README.md [hunk 4]
```diff
@@ -327,6 +327,8 @@ symbol = "🎁 "
 ## Python
 
 The `python` module shows the currently installed version of Python.
+It will also show the current Python virtual environment if one is
```
Comment (matchai) [inline docs/config/README.md]: 👍

**Comment by matchai** [comment]
@all-contributors please add @MaT1g3R for code, docs, and test.

**Pull request merged**